docs/3.x.x/getting-started/quick-start.md

```diff
@@ -1,285 +1,281 @@
 
-***
+   Now let's add **Contemporary**.
 
-## 5. Consume the API
+   Click the **+ Add New Category** button to add the second category **Contemporary**.
 
-Your API is now ready and [contains data](#_4-add-content). At this point, you'll probably want to use this data in mobile or desktop applications.
-In order to do so, you'll need to allow access to other users (identified as 'Public').
+   Type **"Contemporary"** into the **Name** field.
 
-**1 -** Go to the [**Roles & Permissions View**](http://localhost:1337/admin/plugins/users-permissions/roles) by clicking on **Roles & Permissions** link in the left menu.
+   You will see **Restaurants (0)** to the right. Select **Strapi Restaurant** to add this category to the restaurant.
 
-![Auth & Permissions - Home](../assets/getting-started_manage_role_home.png)
+   And then press the **Save** button.
 
-**2 -** Click on the `Public` role, enable the actions related to your new Content Type and save:
+You return to the **Category** Content Type page. You see both categories listed. Both have been assigned to the **Restaurant** we created earlier.
 
-![Auth & Permissions - Edit Public](../assets/getting-started_allow_access.png)
+::: warning NOTE
+
+If you want to add Categories directly from the **Restaurants** Content Type, you simply click on the Restaurant and add, edit or change **EXISTING** categories. Otherwise, you can create and add new **Categories** from the **Category Content Type** as we did above.
 
-::: note
-You should now be able to get the list of posts from the API: [http://localhost:1337/posts](http://localhost:1337/posts). 
 :::
 
-### List entries (GET)
+The next steps involve [setting roles and permissions](#_5-set-roles-and-permissions) for these content types. Let's do that.
 
-To retrieve the list of posts, use the `GET /posts` route.
+Click on the Strapi Logo to return to the main administration panel.
 
-Generated APIs provide a handy way to filter and order queries. In that way, ordering posts by price is as easy as `GET http://localhost:1337/posts?_sort=price:asc`. For more informations, read the [filters documentation](../guides/filters.md).
+---
 
-Here is an example using Axios:
+## 5. Set roles and permissions
 
-```js
-import axios from 'axios';
+By default, Strapi publishes all Content Types with restricted permissions. Which means you have to explicitly give permissions to each Content Type you create.
 
-// Request API.
-axios
-  .get('http://localhost:1337/posts', {
-    params: {
-      _sort: 'createdAt:desc' // Generates http://localhost:1337/posts?_sort=createdAt:desc
-    }
-  })
-  .then(response => {
-    // Handle success.
-    console.log('Well done, here is the list of posts: ', response.data);
-  })
-  .catch(error => {
-    // Handle error.
-    console.log('An error occurred:', error);
-  });
-```
+1. We are going to give **Public** (any web browser with the correct link) access to the **Restaurant** Content Type.
```

Comment:
Add gif : Managing Access permissions and fetching API : https://drive.google.com/drive/folders/19h91Z5hUXTzsEhyZfhFiS9EQVSlorlGc

Reply:
N/A - we now have the videos for each section.